api/src/test/java/org/openmrs/api/CohortServiceTest.java

```diff
@@ -24,6 +24,7 @@
 import java.util.Collection;
 import java.util.Date;
 import java.util.List;
+import java.util.Set;
```

Comment:
Is this import used somewhere?

Reply:
yeah, it's using.

Reply:
where?

Reply:
Sorry, I'm wrong i was looked in the Cohort.java file. let me remove that line and update this.

Reply:
Updated!